.circleci/config.yml

```diff
@@ -20,4 +22,19 @@ jobs:
               echo "Awesome! GCS integration tests are enabled."
             fi
       - run: make format
-      - run: make test
\ No newline at end of file
+      - run: make test
+      - run: make docker
+      - run:
+          name: Run tests on generated Docker image
+          command: |
+            docker run improbable/thanos --help
```

Comment:
Why is that for? Also if `ruler` is missing

Reply:
Hey @Bplotka, I agree for the time being this is a bit overkill as the all the Dockerfile is doing is dropping the binary in. However I felt I should include this so at least we have some rudimentary image tests going forwards if we make any Dockerfile changes. WDYT?

Reply:
Yea, but what are we testing here?
Just if dockerfile can build and run thanos binary? :+1: 

 If that is the case maybe `docker run improbable/thanos --help` is enough here?

Reply:
Yes, just testing the docker image that has been build runs as expected for the different setups before pushing to repo.